website/docs/d/kubernetes_cluster.html.markdown

```diff
@@ -172,6 +174,14 @@ An `azure_active_directory_role_based_access_control` block exports the followin
 
 ---
 
+A `bootstrap_profile` block exports the following:
+
+* `artifact_source` - The source from which artifacts are pulled during bootstrap. Possible values include `Direct` and `Cache`.
```

Comment:
do we need `possible values` for a data source?

Reply:
Updated.